test/verify/check-metrics

```diff
@@ -1525,6 +1525,10 @@ class TestGrafanaClient(testlib.MachineCase):
             except RuntimeError as e:
                 if "Failed to fetch plugins from catalog" not in str(e):
                     raise
+            except testlib.Error as e:
+                if not e.msg.startswith("timeout"):
```

Comment:
I wonder if you could make the thing we're matching on a bit longer.  Is there also some way to check if we actually got the catalog message logged somewhere?

Reply:
This hack will disappear tomorrow, I'm not overly concerned about improving it TBH.

Reply:
See #19669